data_collection/gazette/spiders/ba_conde.py

```diff
@@ -0,0 +1,7 @@
+from gazette.spiders.base.portal_da_transparencia import PortalDaTransparenciaBaseSpider
+
+
+class BaCondeSpider(PortalDaTransparenciaBaseSpider):
+    TERRITORY_ID = "2908606"
+    name = "ba_conde"
+    start_urls = ["http://ba.portaldatransparencia.com.br/prefeitura/conde/"]
```

Comment:
The last edition dates from 30/Dezembro/2020. Could you take a look at the current gazettes' website?

Reply:
It's here now: http://botupora.ba.gov.br/diario_oficial
Underneath, they seem to be a platform called http://procedebahia.com.br (if there are other spiders on top of this platform, it may be possible to reuse code)